lib/ember-test-helpers/test-module.js

```diff
@@ -230,6 +230,13 @@ export default Klass.extend({
     this.container = items.container;
     this.registry = items.registry;
 
+
+    var viewRegistry = this.container.lookup('-view-registry:main');
+    if (viewRegistry) {
+      var View = Ember.__loader.require('ember-views/views/view').default;
+      View.views = viewRegistry;
```

Comment:
No, this was necessary. Ember is internally using the View.views list still, and it only copies that list out of the container during app instance initialization.

Our integration tests don't go down that initialization path.